test/integration/alias_execution_test.exs

```diff
@@ -10,51 +10,51 @@ defmodule Integration.AliasExecutionTest do
 
   test "alias executes properly", %{user: user} do
     send_message(user, "@bot: operable:alias new my-alias \"echo my alias\"")
-    response = send_message(user, "@bot: my-alias")
+    response = send_message(user, "@bot: my-alias") |> Map.fetch!("response")
```

Comment:
What else is in the response that comes back? Can we fold this `Map.fetch!/2` call into the implementation of `send_message/2`?